controllers/disruption_controller_test.go

```diff
@@ -190,6 +191,27 @@ var _ = Describe("Disruption Controller", func() {
 		})
 	})
 
+	Context("disruption expires naturally", func() {
+		BeforeEach(func() {
+			disruption.Spec.Count = &intstr.IntOrString{Type: intstr.String, StrVal: "100%"}
+			disruption.Spec.DurationSeconds = int64(timeout.Seconds() + 5)
```

Comment:
This test adds like ~25 seconds to the unit tests. Worth it?

Reply:
You can let it here, it'll be modified with e2e tests (which will take more time anyway as they really go through injections and cleanups).